.ci/docker/node-container/Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /npm
 RUN ( [ "${NODE_VERSION%%.*}" -le 8  ] \
```

Comment:
`-le 8` also includes the latest version of Node.js 8 right? I think we intended it only to include Node.js 8.1.0 or below. While not technically related to this PR I just wanted to point it out as this fix will then also affect the latest version of Node.js in that case. But in that case, feel free to change it in another PR if that's easier 😃

Reply:
we try to keep it simpler, to check <8.1 make it a little more complex and IMHO it not worth

Reply:
Taking a second look at this, isn't `NODE_VERSION` hardcoded to be `8` in this Dockerfile btw?

Reply:
no, is not, it is the default value if you do not pass a `--build-arg NODE_VERSION=42` to the `docker build` command